modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -112,6 +116,34 @@ This requires an Enterprise license.
 This mode is not recommended for production clusters. Only set to `off` if you need to move partitions manually.
 |===
 
+== Intra-broker partition balancing
+
+In Redpanda, every partition replica is assigned to a core on a broker. While Redpanda's default <<partition-replica-balancing,partition balancing>> monitors cluster-level events such as the addition of new brokers or broker failure to balance partition assignments, it does not account for the distribution of partitions _within_ an individual broker. That means that some cores on a broker may inadvertently host many partitions of heavily used topics and cause the CPU to be xref:manage:monitoring.adoc#cpu-usage[overburdened].
```

Comment:
@ztlpn are there other metrics or other things users could do to monitor and then determine whether they should manually trigger a rebalance?

Reply:
That would be the `vectorized_reactor_utilization` metric. Here are some relevant internal docs: https://redpandadata.atlassian.net/wiki/x/EYC7Gw (CPU section)